### ISSUE TITLE
Having multiple `push_notebook` calls in the same widget callback doesn't work

### ISSUE THREAD

**Issue by aiguofer** [bokeh/bokeh#5415]
When using Jupyter notebooks with multiple notebook handlers, calling `push_notebook` for more than 1 handler in a widget callback causes the following error:

```
Exception in Comm callback Error: event model wasn't known(…) Error: event model wasn't known
    at t.apply_json_patch (https://cdn.pydata.org/bokeh/release/bokeh-0.12.3.min.js:9:10337)
    at t._ (https://cdn.pydata.org/bokeh/release/bokeh-0.12.3.min.js:9:12596)
    at Comm._callback (http://localhost:8888/static/notebook/js/main.min.js?v=e80a0ad9bee7ad0d5865624902d5405d:22110:17)
    at Comm.handle_msg (http://localhost:8888/static/notebook/js/main.min.js?v=e80a0ad9bee7ad0d5865624902d5405d:22118:14)
    at http://localhost:8888/static/notebook/js/main.min.js?v=e80a0ad9bee7ad0d5865624902d5405d:22046:22 Object
```

See this [gist](https://gist.github.com/aiguofer/ad41c3539407d4a8b0c152dc62cd059d) for a reproducible example

The workaround for now is to plot everything using `layout` in a single `show` call.

**Comment by bryevdv** [comment]
Marking as a `bug` but it's not immediately clear offhand whether this will be supportable or not.

**Comment by bryevdv** [comment]
This appears to be fixed by #6945
